.prettierrc.json

```diff
@@ -0,0 +1,5 @@
+{
+  "bracketSpacing": false,
```

Comment:
Shouldn't need this.  This is what the `.prettierrc.js` is supposed to do above.  If we need to make changes, we do it in `gts/.prettierrc.json` ideally.

Reply:
if this line is remove, all bracket space will be brought back, but I remember we do want to remove the extra space to align with some google style? 
if that is the case, I will make the change to gts/.prettierrc.json

Reply:
@xiaozhenliu-gg5 I was under impression it's _already_ in gts defaults :) Is it?

Reply:
yea, I had the same impression, but if it is default as false, removing this line should not affect the linter format, right?

Reply:
just checked `gts/.prettierrc.json`, `singleQuote` is set to be true as default, but if I remove the `singleQuote: true` line, double quotes will replace single quotes. I got the feeling that if client specify a custom `.prettierrc.json` which will override the default one?

Reply:
removed custom `.prettierrc.json` problem solved. thank you!